### ISSUE TITLE
Add to Paradox Mods

### ISSUE THREAD

**Issue by LutherZhao** [Mbyron26/PlayerMoneyController#4]
Hi, thank you for this awesome mod. Any chance you can add it to the official Paradox Mods?

**Comment by Bandar76** [comment]
Yes, I miss this mod very much and I have not found anything like it in paradox mods. I hope to add it
